tables/isolate_mutations.csv

```diff
@@ -10332,3 +10332,10 @@ B.1.1.529 Spike,S,856,K,NULL,NULL
 B.1.1.529 Spike,S,954,H,NULL,NULL
 B.1.1.529 Spike,S,969,K,NULL,NULL
 B.1.1.529 Spike,S,981,F,NULL,NULL
+Chen21e-B.1.1.7,S,69,del,NULL,NULL
+Chen21e-B.1.1.7,S,70,del,NULL,NULL
+Chen21e-B.1.1.7,S,144,del,NULL,NULL
+Chen21e-B.1.1.7,S,145,del,NULL,NULL
+Chen21e-B.1.351,S,242,del,NULL,NULL
+Chen21e-B.1.351,S,243,del,NULL,NULL
+Chen21e-B.1.351,S,244,del,NULL,NULL

```

Comment:
Please also add WA1+S:614G, WA1+S:501Y+614G, Chen21e-B.1.1.28, Chen21e-B.1.429 and Chen21e-B.1.617.1. The whole mutation lists can be found from supplementary table 1.

Reply:
Btw, you have to use the conversion table to get the correct gene and position numbers: https://github.com/hivdb/covid-drdb-payload/wiki/Variants,-isolates-and-mutations#sars-cov-2-genome--gene-position-conversion-table

Reply:
I need a little guidance on this as I tried to explore the S1 file but could not decide what to add from there.

Reply:
Of course. Here are some examples using table S1.

<img width="761" alt="Screen Shot 2022-01-21 at 10 19 01 AM" src="https://user-images.githubusercontent.com/809865/150584479-c7676dee-f4f5-472f-a7fd-48e879846ae6.png">

<img width="1263" alt="Screen Shot 2022-01-21 at 10 57 55 AM" src="https://user-images.githubusercontent.com/809865/150584528-b9a97ece-4479-4fbb-8dcc-dfdf67bb0a5e.png">

Noted that the ORF1ab offset is from https://github.com/hivdb/covid-drdb-payload/wiki/Variants,-isolates-and-mutations#sars-cov-2-genome--gene-position-conversion-table, column "Alt AA position".

Also attach the table S1 I updated:

[41586_2021_3720_MOESM2_ESM (3).xlsx](https://github.com/hivdb/covid-drdb-payload/files/7916019/41586_2021_3720_MOESM2_ESM.3.xlsx)

Reply:
Need some time to go through this table & links. Thanks for sharing it.